### PR TITLE
Base: Add a stack of quotes I collected :^)

### PR DESCRIPTION
**Pull request by kleinesfilmroellchen**

### CODE REVIEW

**Review by Hendiadyoin1 — APPROVED** [review]

File: Base/res/fortunes.json [hunk 1]
```diff
@@ -114,5 +114,80 @@
...
+    "utc_time": 1640122419
+  },
+  {
+    "quote": "why is german",
+    "author": "sin-ack",
+    "url": "https://discord.com/channels/830522505605283862/830525235803586570/917781310766993408",
+    "utc_time": 1638886516
+  },
+  {
+    "quote": "\n> Lasciate ogni speranza, voi ch'entrate!\n> Abandon hope all ye who enter here\n~ Dante",
```
Comment (linusg) [inline Base/res/fortunes.json:150]: is the \n at the beginning on purpose?

File: Base/res/fortunes.json [hunk 2]
```diff
@@ -114,5 +114,80 @@
...
+    "context": "https://discord.com/channels/830522505605283862/830739873119207426/895421023833436160"
+  },
+  {
+    "quote": "and then the firebug nation attacked",
+    "author": "gggggg",
+    "url": "https://discord.com/channels/830522505605283862/830525031720943627/893862331312902166",
+    "utc_time": 1633183786
+  },
+  {
+    "quote": "there is no need to narrate all your random testing :)",
```
Comment (linusg) [inline Base/res/fortunes.json:176]: not sure I'd include this one, it's oddly unexciting

File: Base/res/fortunes.json (hunk 1 above)
Comment (kleinesfilmroellchen) [reply Base/res/fortunes.json:150]: No, I'll remove it

File: Base/res/fortunes.json (hunk 2 above)
Comment (kleinesfilmroellchen) [reply Base/res/fortunes.json:176]: sure

**Pull request merged**